src/style.css

```
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+}
+
+.stars__item {
+  display: inline-block;
+  margin: 0 1px;
+  height: 12px;
+  width: 13px;
+  content: url("images/star.svg");
+}
+
+.stars--1 .stars__item:nth-child(1) {
+  content: url("images/star-active.svg");
+}
+
+.stars--2 .stars__item:nth-child(-n+2) {
+  content: url("images/star-active.svg");
+}
+
+.stars--3 .stars__item:nth-child(-n+3) {
+  content: url("images/star-active.svg");
+}
+
+.stars--4 .stars__item:nth-child(-n+4) {
+  content: url("images/star-active.svg");
+}
+
+.stars--5 .stars__item:nth-child(-n+5) {
```

```suggestion
.stars--5 .stars__item {
```
since there is only five stars, this is useless